packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -0,0 +1,152 @@
+import Action from "tasit-action";
+const { ConfigLoader, ERC20 } = Action;
+const { DetailedERC20 } = ERC20;
+import { ethers } from "ethers";
+import { expect } from "chai";
+import GnosisSafe from "./GnosisSafe";
+import { local as localAddresses } from "../../tasit-contracts/3rd-parties/gnosis/addresses";
+import { createFromPrivateKey } from "../../tasit-account/dist/testHelpers/helpers";
+import {
+  mineBlocks,
+  createSnapshot,
+  revertFromSnapshot,
+  confirmBalances,
+} from "tasit-action/dist/testHelpers/helpers";
+
+// The goal of this integration test suite is to use only exposed classes
+// from TasitSdk. ProviderFactory is used here as an exception
+// as the clearest way to get a provider
+// in this test suite. Eventually, maybe ProviderFactory may move to
+// some shared helper dir.
+import ProviderFactory from "tasit-action/dist/ProviderFactory";
+
+const { GnosisSafe: GNOSIS_SAFE_ADDRESS } = localAddresses;
+const ERC20_ADDRESS = "0x37E1A58dD465D33263D00185D065Ee36DD34CDb4";
+
+const { utils: ethersUtils } = ethers;
+const { bigNumberify } = ethersUtils;
+
+const ZERO = 0;
+const ONE = bigNumberify(`${1e18}`);
+
+// 100 gwei
+const GAS_PRICE = bigNumberify(`${1e11}`);
+
+describe("GnosisSafe", () => {
+  const config = {
```

Comment:
Can we import this test config from somewhere shared?

Reply:
Yes, I've moved that to `mochaSetup.js` file on my last commit but I can improve that exposing the config from one place (action helper) for all other packages. Make sense?

Reply:
Great. Yep, makes sense! Thanks.

Reply:
Done.

Reply:
Great, thanks. Resolving this!